src/message.ts

```diff
@@ -414,6 +414,59 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   *
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   *
+   * @example
+   * ```ts
+   * const content = message.content()
+   * const contactList = message.mention()
+   * console.log(content)
+   * console.log(contactList)
+   * ```
+   */
+  public mention(): Contact[] {
+    let contactList: Contact[] = []
+    const room = this.room()
+    if (this.type() !== MsgType.TEXT || !room ) {
+      return contactList
+    }
+
+    const atStringList = this.content().match(/@\S+ ?/g)
+    if (!atStringList) return contactList
+
+    const mentionList = atStringList.map(element => {
+      /**
+       * `fake@` and `real@` definition
+       * Supposed a wechat contact called `lijiarui`
+       * `fake@` means @ event is produced by typing '@lijiarui ' (mock mention behaviour by web wechat)
+       * `real@` means @ event is produced by long press the contact's avatar (the real behaviour in cellphone)
+       *
+       * `fake@` return element.slice(1, -1), `real@` return element.slice(1)
+       */
+      return element.slice(1)
+    })
```

Comment:
@xinbenlv It's just my personal favorite.

And I'm also supporting to not use semicolons, because of:
1. <http://blog.izs.me/post/2353458699/an-open-letter-to-javascript-leaders-regarding>
1. <http://inimino.org/~inimino/blog/javascript_semicolons>
1. <https://www.youtube.com/watch?v=gsfbh17Ax9I>

So I would like to suggest all my friends to not use semicolon, which is a _JS Standard Code Style_.
[![JS Standard Coding Style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://standardjs.com/rules.html#semicolons)

> No semicolons.